source/gui/__init__.py

```diff
@@ -75,6 +80,13 @@ def __init__(self):
 				self.Show()
 				self.Hide()
 
+	def _onDestroy(self, evt: wx.EVT_WINDOW_DESTROY):
+		self.Skip(skip=False)  # we should block the destruction events until the sysTrayIcon is destroyed
+		# wx destroys child Windows automatically but `wx.adv.TaskBarIcon` is not a window
+		# so it must be set to be destroyed when destroying our main frame window (#12243)
+		log.debug(f"destroying systray icon")
+		wx.CallAfter(self.sysTrayIcon.Destroy)

```

Comment:
Why is callAfter necessary here?

Reply:
I can't find this in any official documentation, but when doing the original `safeAppExit` PR I found several recommendations in forums to use this method as it queues destruction events safer. Calling `Destroy` directly here causes a crash on the installation process.